### DISCUSSION TITLE
No binary assets in v2024.6.2

### DISCUSSION THREAD

**Discussion by davidolrik** [jdx/mise#2262] [General]
There are no  binary assets in the latest release (v2024.6.2)

Is this on purpose?

**Comment by Marukome0743** [comment]
This appears to have been a temporary delay while the release assets were being published.

The v2024.6.2 release was published at 02:12 UTC, this discussion was created at 07:38 UTC, and the binary assets were uploaded at 13:43 UTC on the same day.

The release currently contains 30 assets, including Linux, macOS, and Windows binaries, archives, and checksum files:

https://github.com/jdx/mise/releases/tag/v2024.6.2

Therefore, the missing assets reported here are now available and no further workaround should be necessary.

*AI-assisted — Tool: Codex; model: OpenAI/GPT-5; version: unavailable.*
